Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/Controller.purs

```diff
@@ -185,13 +185,13 @@ eval (BackPressed flag) state = do
                 continue state
 
 eval (CountDown seconds id status timerID) state = do
-        _ <- pure $ printLog "timer" seconds
+        _ <- pure $ printLog "timer" $ show seconds
         if status == "EXPIRED" then do
             _ <- pure $ clearTimer timerID
-            let newState = state{data{timer = "", timerID = ""},props = state.props{resendEnable = true}}
+            let newState = state{data{timer = 15, timerID = ""},props = state.props{resendEnable = true}}
```

Comment:
why timer is reset to 15 here? is this the reset value?

Reply:
Timer has been updated to 30 seconds.